apps/sms/js/thread_list_ui.js

```diff
@@ -857,13 +846,7 @@ var ThreadListUI = {
     var threadUITime = threadUINode ? +threadUINode.dataset.time : NaN;
     var recordTime = +thread.timestamp;
 
-    // For legitimate in-memory thread objects, update the stored
-    // Thread instance with the newest data. This check prevents
-    // draft objects from inadvertently creating bogus thread
-    // objects.
-    if (Threads.has(thread.id)) {
-      Threads.set(thread.id, thread);
-    }
+    Threads.set(thread.id, thread);
```

Comment:
Since we will store all the threads(including draft) into `Threads`, I remove this checking to prevent the draft thread missing in threadlist view. Or you think it's more appropiate to set the draft into Threads while in saveDraft?